spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'rails_helper'
 
 RSpec.describe V0::SessionsController, type: :controller do
-  let(:mvi_user) { FactoryGirl.build(:mvi_user) }
+  let(:mvi_user) { FactoryGirl.build(:loa3_user) }
```

Comment:
I wouldn't _build_ a `loa3_user` and call it `mvi_user`.  Each example will use this user in a different way - so each example that used to use `mvi_user` would need to be updated individually.  

This rspec is an area of tech debt because the sessions_controller has too many responsibilities at the moment.